integration-tx-api/src/main/java/io/seata/integration/tx/api/util/ProxyUtil.java

```diff
@@ -37,6 +37,21 @@ public static <T> T createProxy(T target) {
         return createProxy(target, target.getClass().getName());
     }
 
+    /**
+     * The API for generating proxy for target. It can be used by spring aop, or
+     * provide user to generate proxy manually.
+     * <p>
+     * At TM side, It can be used for the target bean with @GlobalTransactional or @GlobalLock to generate proxy which start global transaction.
+     * At RM side, if you use TCC mode， It can be for target bean with @TwoPhaseBusinessAction to generate proxy which register branch source
```

Comment:
```suggestion
     * At RM side, if you use TCC mode, It can be for target bean with @TwoPhaseBusinessAction to generate proxy which register branch source
```